.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+on:
+  pull_request:
+  push: { branches: master }
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        ruby-version: [2.3.8, 2.4.10, 2.5.9, 2.6.6, 2.7.6, 3.0.4, 3.1.2]
```

I recommend not to specify minor versions unless there is a special reason. 
```suggestion
        ruby-version: ['2.3', '2.4', '2.5', '2.6', '2.7', '3.0', '3.1']
```